### PR TITLE
Adding SIG Process/Workflow Guide and related images

### PR DESCRIPTION
**Pull request by LappleApple**
This PR creates documentation to support SIGs in their efforts to improve workflow and processes. It supports the Program Manager role proposed in [#5563](https://github.com/kubernetes/community/pull/5563) but can be useful to any SIG lead or contributor.

/assign @saschagrunert @LappleApple @nikhita
cc: @kubernetes/sig-release-leads @kubernetes/sig-contributor-experience-leads @kubernetes/steering-committee

### CODE REVIEW

**Comment by LappleApple** [comment]
sig/release
sig/contributor-experience

**Comment by dims** [comment]
@ehashman @saschagrunert @SergeyKanzhelev - since Lauri with her PM hat has extensively helped your SIGs. Can you please chime in on this PR? Does this cover all things that helped you? Does this have enough info for a brand new person (not Lauri) to aspire for and work towards in other SIGs?

**Comment by dims** [comment]
@LappleApple Do you mind splitting the governance pieces into a separate PR? ( committee-steering/governance/sig-governance.md + contributors/chairs-and-techleads/program-manager.md )

This would help the steering focus on just the governance aspect and new role without worrying about the guide which may be better reviewed by leads@ etc.

**Comment by LappleApple** [comment]
Here's the new PR with just the PM governance docs: [#5566](https://github.com/kubernetes/community/pull/5566). 

Thanks for helping me with this PR split, @dims!

**Review by ehashman — COMMENTED** [review]
Instrumentation/Node bits LGTM

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+## Triage and Triaging Tools
+
+Triaging is an effective method for improving SIG responsiveness to contributor requests—and, consequently, keeping contributors engaged and motivated. It’s also useful for:
+- keeping on top of incoming work and delegating it to contributors
+- maintaining company agnosticism. “Devs from other SIGs don’t have to ask for any favors in getting PRs to move forward—it’s just part of the process,” API Machinery Chair Fede Bongiovanni says. “Features don't get biased development.”
+
+You can find [comprehensive triaging guidelines](https://github.com/kubernetes/community/blob/c78c26294b8bd137dd5cc0c807aadc728dad6a57/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
+- **Gubernator**: a personal dashboard that shows you PRs awaiting your review
+- **Triage Party**: a stateless web app to optimize issue and PR triage for large open-source projects using the GitHub API. More details are below. (Cluster Lifecycle, SIG Release) 
+- **GitHub project boards**: these kanban boards allow you to track GitHub issues and PRs in an automated way. More details are below.
```
[the remark this answers is not in the document]
Comment (LappleApple) [reply contributors/chairs-and-techleads/process-workflow-guide.md]: Good suggestion -- just added links

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+- Contributors who want to support SIGs with process and workflow improvements
+- Contributors who want to become SIG leads or drive Kubernetes Enhancements Proposals (KEPs) and projects
+
+## Why This Exists
+
+The primary reason is to reduce risk. Without processes in place to engage and graduate contributors, keep work flowing forward to completion, delegate, and ensure that the highest-value work gets done, Kubernetes will lose contributors, momentum, and growth opportunities. And you, as leads, might burn out from trying to do it all. Process assists in delegation, which is how overworked and overstretched SIG leads can load-balance. 
+
+The idea here is progress, not perfection. Please try to put past process experiments and things-that-didn’t-work behind you. What failed yesterday might work today. You might need to recalibrate your idea of “failure”—a small bit of progress is still progress. Also, from failures we learn.
+
+You might also take a cue from SIG Release and create a Program Manager role for your SIG. Reach out to @lauri if you’d like to explore this option.
```
Comment (saschagrunert) [inline contributors/chairs-and-techleads/process-workflow-guide.md:17]:
```suggestion
You might also take a cue from SIG Release and create a Program Manager role for your SIG. Reach out to [@LappleApple](https://github.com/lappleapple) if you’d like to explore this option.
```

**Review by saschagrunert — APPROVED** [review]
LGTM, just one nit

**Comment by LappleApple** [comment]
Thanks for the feedback, everyone. I'm planning to create a simple A4-size diagram to make this content more insta-usable/digestible.

**Comment by neolit123** [comment]
LGTM, i think this guide can help SIGs organize better.

**Comment by nikhita** [comment]
cc @kubernetes/steering-committee

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
+# Process and Workflow Guide
+
+Author: **[@LappleApple](https://github.com/lappleapple)** || Contributors: **[@annajung](https://github.com/annajung)**, **[@kikisdeliveryservice](https://github.com/kikisdeliveryservice)**, **[@neolit123](https://github.com/neolit123)** 
+
+## Who This Is For
+
+- SIG leads who want guidance on how to structure workflow and processes using models explored successfully by other SIGs
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:7]:
```suggestion
- Leads who want guidance on how to structure workflow and processes using models explored successfully by other groups 
```
Might be nitty, but SIGs are our main group, but this doc would be applicable for WGs and subproject. Language would need to be updated throughout the doc though.

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+  - [When You Need More Help with KEPs/KEP Process](#when-you-need-more-help-with-kepskep-process)
+- [Creating Roadmaps](#creating-roadmaps)
+  - [Example: Roadmapping Templates for a Release Cycle](#example-roadmapping-templates-for-a-release-cycle-sigs-docs-release) (SIGs Docs, Release)
+  - [Example: Brainstorming Template for Creating a Roadmap](#example-brainstorming-template-for-creating-a-roadmap-sig-releaserelease-engineering) (SIG Release/Release Engineering)
+  - [Example: Umbrella Issues for Roadmapping an Initiative/Epic](#example-umbrella-issues-for-roadmapping-an-initiativeepic-sigs-testing-release) (SIGs Testing, Release)
+- [Managing Communications: Delegating and Driving Transparency](#managing-communications-delegating-and-driving-transparency)
+  - [Slack](#slack)
+  - [Mailing List and GitHub Notification Best Practices](#mailing-list-and-github-notification-best-practices) (SIG Contributor Experience)
+
+For brevity’s sake, not every aspect of running a SIG is covered. If you have questions, please reach out in **#chairs-and-leads**.
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:52]:
```suggestion
For brevity’s sake, not every aspect of running a SIG is covered. If you have questions, please reach out in [#chairs-and-leads](https://kubernetes.slack.com/messages/chairs-and-techleads).
```

Added link 👍

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
+## Triage and Triaging Tools
+
+Triaging is an effective method for improving SIG responsiveness to contributor requests—and, consequently, keeping contributors engaged and motivated. It’s also useful for:
+- keeping on top of incoming work and delegating it to contributors
+- maintaining company agnosticism. “Devs from other SIGs don’t have to ask for any favors in getting PRs to move forward—it’s just part of the process,” API Machinery Chair Fede Bongiovanni says. “Features don't get biased development.”
+
+You can find [comprehensive triaging guidelines](https://github.com/kubernetes/community/blob/c78c26294b8bd137dd5cc0c807aadc728dad6a57/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:82]:
```suggestion
You can find [comprehensive triaging guidelines](/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
```

For links to items in the same repo, we prefer to use the path (also not linking to a specific commit)

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+- **[Triage Party](https://github.com/google/triage-party)**: a stateless web app to optimize issue and PR triage for large open-source projects using the GitHub API. More details are below. (Cluster Lifecycle, SIG Release) 
+- **[GitHub project boards](https://docs.github.com/en/github/managing-your-work-on-github/managing-project-boards)**: these kanban boards allow you to track GitHub issues and PRs in an automated way. More details are below.
+
+### Triage Meetings
+
+Triaging is highly recommended for improving workflow and responsiveness to contributors. 
+
+Some SIGs, like API Machinery and [Instrumentation](https://docs.google.com/document/d/1FE4AQ8B49fYbKhfg4Tx0cui1V0eI4o3PxoqQPUwNEiU/edit), host triage-specific meetings to manage their backlogs. If your SIG’s backlog is small, however, then you can triage and refine during your bimonthly meetings by setting aside time. In November 2020 we rolled out a needs-triage label to support triage activities. You can add your [user feedback about the label here](https://github.com/kubernetes/community/issues/5299#issuecomment-776914413).
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:91]:
```suggestion
Some SIGs, like API Machinery and [Instrumentation](https://docs.google.com/document/d/1FE4AQ8B49fYbKhfg4Tx0cui1V0eI4o3PxoqQPUwNEiU/edit), host triage-specific meetings to manage their backlogs. If your SIG’s backlog is small, however, then you can triage and refine during your bimonthly meetings by setting aside time. In November 2020 we rolled out a needs-triage label to support triage activities. You can add your [user feedback about the label here](https://github.com/kubernetes/community/issues/5299).
```

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+
+#### Example: Triage Party for Skaffold
+
+[Triage Party](https://github.com/google/triage-party) is an open-source tool that focuses on “reducing response latency for incoming GitHub issues and PR's, and ensure[s] that conversations are not lost in the ether.” Project creator Thomas Strömberg reports that using Triage Party for Minikube and Skaffold workflows has led to dramatic improvements in responsiveness to contributor requests. By “dramatic,” he means *40 times faster*.
+
+<img src="triage_party_skaffold.png" width="60%" height="60%">
+
+**Other Uses of Triage Party**:
+- SIG CLI uses it to manage [kubectl](https://tp.kuberneddies.dev/s/daily) workflow, including quarterly bug scrubs and daily/weekly triage
+- SIG Release has set up a Triage Party instance and will begin using it for bug triage, issue triage, and other activities in 1.21.
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:109]: Is this live now?

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 8]
```diff
@@ -0,0 +1,253 @@
...
+
+#### Managing KEP Scope
+
+Breaking down large KEPs into smaller increments of value helps you to ship faster and more frequently, and creates shorter feedback loops. 
+
+Network and Storage are two SIGs that regularly break down large KEPs to ship more predictably. An example from Storage is volume health: The alpha version was a very small feature that could be expanded in later cycles. When it’s not possible or simple to break down KEPs, consider holding them until the next release cycle to avoid missing the Code Freeze deadline.
+
+#### When You Need More Help with KEPs/KEP Process
+
+Please ping the Enhancements subproject at **#enhancements** with your questions, needs, and concerns. In addition to having extensive knowledge about the KEP process and how to manage it, the subproject also includes several product manager-contributors who will embed with your SIG and help you with prioritization, user story creation for KEPs, roadmapping, and other efforts that can save you time and reduce complexity.
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:195]:
```suggestion
Please ping the Enhancements subproject at [#enhancements](https://kubernetes.slack.com/messages/enhancements) with your questions, needs, and concerns. In addition to having extensive knowledge about the KEP process and how to manage it, the subproject also includes several product manager-contributors who will embed with your SIG and help you with prioritization, user story creation for KEPs, roadmapping, and other efforts that can save you time and reduce complexity.
```

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 9]
```diff
@@ -0,0 +1,253 @@
...
+
+### Example: Brainstorming Template for Creating a Roadmap (SIG Release/Release Engineering)
+
+Release Manager Adolfo “Puerco” García Veytia and SIG Release Tech Lead Dan Mangum used Miro to brainstorm a roadmap for creating a bill of materials:
+
+<img src="BOM board.png">
+
+SIG Release/Release Engineering is using [this template](https://docs.google.com/document/d/1U-SxIQKwEDmlj3QMxmKUl2laFRKaelDwBBBqWtnxPvw/edit) to draft a 2021 North Star vision that tells a coherent story to the project and contributors about what work is important this year, and why.
+
+SIG Docs also does quarterly planning and objectives... (**find out more**)
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:221]:
```suggestion
SIG Docs also does quarterly planning and objectives... (**find out more**)
```
is this a TODO for the future?

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+### Slack
+
+- Use direct messages only for sensitive topics (personal, HR-related).  
+- When someone pings via DM about work or a question that could be discussed/answered by others, direct them to your public channel.
+- Use 2-3 channels as your main go-to for discussion.
+- Create team handles and use them to draw attention to time-sensitive or workflow-related Slack messages (posted GitHub issues, PRs). This can drive delegation and discussion.
+
+### Mailing List and GitHub Notification Best Practices
+
+Visit /community for [documentation on this topic](https://github.com/kubernetes/community/blob/master/communication/best-practices.md).
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:253]:
```suggestion
See the [communication best practices for mailing list and GitHub notifications](/communication/best-practices.md).
```

The doc is in k/community already 👍

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks for putting this together \o/
A few small suggestions and one general thing, please don't use spaces in the filenames. Something like `_` or `-` is preferred or removing the space altogether.

File: contributors/chairs-and-techleads/process-workflow-guide.md (hunk 7 above)
Comment (LappleApple) [reply contributors/chairs-and-techleads/process-workflow-guide.md:109]: Actually yes, it went live a couple of weeks ago: https://release.triage.k8s.io/s/milestone. I'll add that link.

File: contributors/chairs-and-techleads/process-workflow-guide.md (hunk 9 above)
Comment (LappleApple) [reply contributors/chairs-and-techleads/process-workflow-guide.md]: Oops --removed

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+- [Triage Meetings](#triage-meetings) (SIGs API Machinery, Instrumentation)
+  - [Staffing Triage Activities](#staffing-triage-activities) (SIG Docs)
+- [Triage Party](#triage-party) (SIGs Cluster Lifecycle, CLI, Release)
+  - [Example: Triage Party for Skaffold](#example-triage-party-for-skaffold) (SIG Cluster Lifecycle)
+- [GitHub Project Boards](#github-project-boards) (SIGs Node, Windows, Release) 
+  - [Board Set-up Basics](#board-set-up-basics)
+  - [Example: Pull Request Triage Board](#example-pull-request-triage-board-sig-node) (SIG Node)
+  - [Example: Board Highlighting Risks](#example-board-highlighting-risks-sig-windows) (SIG Windows) 
+  - [Example: KEP-Specific Issue Triage](#example-kep-specific-issue-triage-sig-release) (SIG Release)
+- Triaging CI Signal Needs (SIG Windows)
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/process-workflow-guide.md:36]: I'm not sure this item exists later on :x

**Review by mrbobbytables — APPROVED** [review]
Thanks!
/approve
/hold
(for any other comments)

File: contributors/chairs-and-techleads/process-workflow-guide.md [hunk 12]
```diff
@@ -0,0 +1,250 @@
...
+
+### Example: SIG Release/Release Engineering
+
+The [Release Engineering](https://github.com/kubernetes/sig-release/tree/master/release-engineering) subproject of SIG Release has published a Ways of Working agreement that covers communication channels and how they’re used; roles and responsibilities; how they discuss work; and other major aspects of workflow and process. 
+
+### Making the Most out of Meetings
+
+Most, if not all, SIGs, WGs, and subprojects already have templates for organizing bimonthly meetings. [Here is a collection of templates](https://miro.com/templates/team-meetings/) for remote meetings that you might also find useful.
+
+You’ll find some SIG-specific meeting tips in Release Engineering’s WoW agreement. A few more:
```
Comment (SergeyKanzhelev) [inline contributors/chairs-and-techleads/process-workflow-guide.md:65]: What is "Release Engineering’s WoW agreement"? Is there a link?

**Review by SergeyKanzhelev — APPROVED** [review]

File: contributors/chairs-and-techleads/process-workflow-guide.md (hunk 12 above)
Comment (LappleApple) [reply contributors/chairs-and-techleads/process-workflow-guide.md:65]: There's a link earlier in the guide -- WoW = "Ways of Working"

Comment (SergeyKanzhelev) [reply contributors/chairs-and-techleads/process-workflow-guide.md:65]: oh, that makes sense. I didn't connect the abbreviation to the previous text. sorry

Comment (LappleApple) [reply contributors/chairs-and-techleads/process-workflow-guide.md:65]: No problem :)

**Comment by parispittman** [comment]
/lgtm
/approve

thanks so much for this Lauri!!! it's awesome to see the examples and how things are in play across the org. will be extremely helpful for new chairs as they onboard.

**Pull request merged**